content/03.methods.md

```diff
@@ -96,67 +96,69 @@ This extension step produces path through the pangenome with consistent coverage
+The Sequence Tube Map has also recently added support for displaying multiple layers or "tracks" of haplotypes or reads[@lancet].
+Together with the GAF support presented here, this allows the user to add several tracks of annotations from indexed GAF files to the Sequence Tube Map as if they were read files, or alongside actual read files.
+The Sequence Tube Map supports assigning a different color or color palette to each track, to facilitate the visualization of different datasets in the same local pangenome region. 
+We have extended this functionality with a new option to use an element's opacity to represent an integer score, like a mapping quality or read coverage.
+For example, one can now visualize the coding regions of genes and coverage tracks for different cell types from the ENCODE project, as described in [Coverage of seven functional datasets from ENCODE] and highlighted in Figure @fig:tubemap.
 
-### Visualization in Bandage
+### Visualization in Bandage-NG
 
-A fork of Bandage[@bandage], called BandageNG[@{https://github.com/asl/BandageNG}], can read GAF files and visualize its paths through the input graph by coloring the nodes.
-Of note, the offset positions are not taken into account, which means that nodes are colored entirely even if the path only overlaps part of it.
-We implemented a wrapper script to facilitate the preprocessing of a subgraph to visualize with BandageNG. 
-It starts by extracting the subgraph of a full pangenome for a region of interest using `vg chunk`.
-This graph is converted to GFA, making sure that the path are written last.
-The script also extracts annotations from indexed files on that same region. 
-Those GAF files are merged and modified if necessary: path names are de-duplicated and paths are trimmed when traversing nodes absent from the extracted subgraph.
-These modifications ensure that the final GAF file can be loaded in BandageNG.
-The outputs of this script can be opened with BandageNG for interactive exploration. 
+Bandage-NG[@{https://github.com/asl/BandageNG}], a tool derived from Bandage[@bandage], can read GAF files and visualize its paths through the input graph by coloring the nodes.
+Of note is that the offset positions are not taken into account, which means that nodes are colored entirely even if the GAF record only overlaps part of it.
+We implemented a wrapper script to facilitate the preprocessing of a subgraph to visualize with Bandage-NG. 
+It starts by extracting the subgraph for a region of interest from a full pangenome using `vg chunk`.
+This graph is converted to GFA, making sure that the paths are written last for compatibility with Bandage-NG.
+The script also extracts annotations relevant to the region of interest from indexed GAF files. 
+The extracted GAF files are merged and modified if necessary: path names are de-duplicated and paths are trimmed<!-- Not split??? --> when traversing nodes absent from the extracted subgraph.
```

Comment:
Here I meant trimmed as in if an end of a path only traverses nodes that are not in the subgraph, I remove them (otherwise it's not accepted by bandage). This occurs for long paths or for paths at the edges of the extracted subgraph. The tails are not included at all so I used "trimmed"

Reply:
Saying "trimmed" suggests to me that it doesn't really contemplate the path leaving the subgraph and then coming back.

Maybe "clipped" is the right word, if it does that? That's what we say is done to the HPRC graphs to get their partial paths.